cmd/controller/checknodehealth/main.go

```diff
@@ -46,11 +48,29 @@ func main() {
 	flag.Parse()
 	defer klog.Flush()
 
+	// Set up controller-runtime logger
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
+
 	klog.InfoS("Starting CheckNodeHealth Controller")
 
+	// Get checker pod image from environment variable
+	checkerImage := os.Getenv("CHECKER_IMAGE")
```

Comment:
Would prefer to rename these variables to `checkerPodImage` and `CHECKER_POD_IMAGE` to make it less confusing what this image is for.